os-package-tracking.zeek

```diff
@@ -156,6 +156,13 @@ function found_package(orig: addr, o: OSInfo, p: PackageVersion, r: RequestInfo)
 # For now - let's be a bit lazy and get the information we want out of the http logging event.
 event HTTP::log_http(i: HTTP::Info)
 	{
+
+	local parts : string_vec;
```

Comment:
Just out of curiosity - is there a reason to move these declarations to the beginning of the event?

Reply:
Ah, yes, there is, ignore this one.